aks/redis/variables.tf

```
@@ -95,6 +95,11 @@ variable "azure_enable_monitoring" {
   default = true
 }
 
+variable "redis_replica_zones" {
```

```suggestion
variable "azure_zones" {
```

I've been naming any Azure-specific variables with `azure_` to keep them consistent.